skimage/draw/tests/test_draw3d.py

```diff
@@ -114,6 +121,94 @@ def test_ellipsoid_levelset():
         ]
     )
 
+    assert_allclose(test, expected)
+    assert_allclose(test_rotation, expected)
+    assert_allclose(test_anisotropic, expected)
+
+
+def test_ellipsoid_rotation_bool():
+    test = ellipsoid(a=2, b=1, c=1, rotation=(np.pi / 2, np.pi / 2, np.pi / 2))
```

Comment:
```suggestion
    test = ellipsoid(a=2, b=1, c=3, rotation=(np.pi / 2, np.pi / 2, np.pi / 2))
```
What do you think about using three distinct lengths here? To make the test more restrictive.

Reply:
I think that the present tests are fine: easy to quickly do back of the envelope verification, the other test parameters have redundancy too